src/core-services/account/resolvers/Group/index.js

```diff
@@ -1,9 +1,11 @@
 import resolveShopFromShopId from "@reactioncommerce/api-utils/graphql/resolveShopFromShopId.js";
 import { encodeGroupOpaqueId } from "../../xforms/id.js";
 import createdBy from "./createdBy.js";
+import createGroup from "./createGroup.js";
 
 export default {
   _id: (node) => encodeGroupOpaqueId(node._id),
   createdBy,
-  shop: resolveShopFromShopId
+  shop: resolveShopFromShopId,
+  createGroup
```

Comment:
`createGroup` shouldn't bein this file. This is the resolver for the `Group` object itself when it appears in a query or a mutation payload.

Reply:
@mikemurray ... I wasnt aware of this but thanks for pointing it out
Done